### PR TITLE
feat(recurrence): Added standups time left indicator

### LINKED ISSUE #7499: Recurrence: Add time left indicator to a recurring meeting
Design: https://github.com/ParabolInc/parabol/issues/7421

AC:
- [ ] when meeting is recurring, time left indicator is visible

### PR DESCRIPTION
**Pull request by BartoszJarocki**
# Description

Fixes #7499 

## Demo

localhost_3000_meet_iTnoRVBsu4_responses
localhost_3000_meet_iTnsw3WyaI_responses

## Testing scenarios

- [ ] start standup meeting, enable recurrence, and check if timer is showing (checking if it works correctly may require changing the default meeting duration [here](https://github.com/ParabolInc/parabol/blob/e938186e0f31105164530844232d0adf0c098c46/packages/server/graphql/public/mutations/startRecurrence.ts#L48), otherwise it might take a while to test 🙃)

## Final checklist

- [x] I checked the [code review guidelines](../docs/codeReview.md)
- [x] I have added [Metrics Representative](../docs/codeReview.md#metrics-representative) as reviewer(s) if my PR invovles metrics/data/analytics related changes
- [x] I have performed a self-review of my code, the same way I'd do it for any other team member
- [x] I have tested all cases I listed in the testing scenarios and I haven't found any issues or regressions
- [x] Whenever I took a non-obvious choice I added a comment explaining why I did it this way
- [x] I added the label `One Review Required` if the PR introduces only minor changes, does not contain any architectural changes or does not introduce any new patterns and I think one review is sufficient'
- [x] PR title is human readable and could be used in changelog

### CODE REVIEW

File: packages/server/graphql/public/mutations/startRecurrence.ts [hunk 1]
```diff
@@ -128,11 +128,11 @@ const startRecurrence: MutationResolvers['startRecurrence'] = async (
     analytics.recurrenceStarted(viewerId, meetingSeries)
   } else {
     const newMeetingSeries = await startNewMeetingSeries(viewerId, teamId, meetingId)
-    dataLoader.get('newMeetings').clear(meetingId)
-
     analytics.recurrenceStarted(viewerId, newMeetingSeries)
   }
 
+  dataLoader.get('newMeetings').clear(meetingId)
```
Comment (BartoszJarocki) [inline packages/server/graphql/public/mutations/startRecurrence.ts:134]: This fixes the issue when the meeting object is stale when restarting the recurrence

**Comment by BartoszJarocki** [comment]
@jmtaber129 may I ask you to take a look? 🙏

File: packages/client/utils/date/relativeDate.ts [hunk 2]
```diff
@@ -19,6 +21,45 @@ interface Opts {
   smallDiff?: string
 }
 
+/**
+ * Creates a human-readable string representing the time till the given date,
+ * for example: 2 days left; 13 hours left; 2 days left, etc or null if the date is in the past
+ * @param date
+ */
+export const humanReadableCountdown = (date: string | Date) => {
```
Comment (jmtaber129) [inline packages/client/utils/date/relativeDate.ts:29]: +2 we should consider using this for timers in other meetings that are more than a few minutes!

**Review by jmtaber129 — APPROVED** [review]
LGTM!

If you'd like, you could also add a tooltip that shows the exact end time (e.g. "Ends at <date time>"), but this is also fine.

**Comment by BartoszJarocki** [comment]
@jmtaber129 great suggestion, will add!

File: packages/client/utils/date/relativeDate.ts (hunk 2 above)
Comment (BartoszJarocki) [reply packages/client/utils/date/relativeDate.ts:29]: good idea, will create a task for it

**Pull request merged**